src/debug.c

```diff
@@ -816,7 +816,8 @@ NULL
         } else {
             addReply(c,shared.ok);
         }
-    } else if (!strcasecmp(c->argv[1]->ptr,"set-skip-checksum-validation") && c->argc == 3) {
+    } else if (!strcasecmp(c->argv[1]->ptr,"set-skip-checksum-validation") &&
+               c->argc == 3) {
```

Comment:
I prefer the older ones, which are over 80, but not too long.

Reply:
im not sure why github shows "} else if (!strcasecmp(c->argv[1]->ptr,"set-skip-checksum-validation") &&" as line that changed... it is same line. I rather not change lines that is not relevant for the pr

Reply:
I've looked at the code, and it has been modified by https://github.com/redis/redis/pull/9357/commits/672827b85ca4b13174afc17ddf8be9abefab9123.
BTW, If the condition in if is multi-line, `{` will be on the next line.

Reply:
I'll change the location of { ... thanks